pom.xml

```diff
@@ -5,6 +5,13 @@
                              http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <!-- This parent is requirements for deploy of artifacts to maven central -->
+    <parent>
+        <groupId>org.sonatype.oss</groupId>
+        <artifactId>oss-parent</artifactId>
+        <version>9</version>
+    </parent>
+
     <groupId>org.checkstyle.autofix</groupId>
```

Comment:
I think we can `org.checkstyle` only for group.
it is ok to have plugins/tools to be just under "checkstyle"

Reply:
In the patch-filter project I see latest master has `<groupId>com.puppycrawl.tools</groupId>`, shall we go with the same groupId? Better to agree on this now while we are still at the start of the project

Reply:
https://repo1.maven.org/maven2/org/checkstyle

Is not existing, so it means it will be a special process to claim ownership of such domain.

Lets use group common for all , as Ruslan suggested 
https://repo1.maven.org/maven2/com/puppycrawl/tools/